### ISSUE TITLE
Question: Unused const in pylandau_src.cpp ?

### ISSUE THREAD

**Issue by leloup314** [SiLab-Bonn/pylandau#22]
The [`const double norm`](https://github.com/SiLab-Bonn/pylandau/blob/master/pyLandau/cpp/pylandau_src.cpp#L151) in `pylandau_src.cpp` is unsused, as far as I can see. Is this a bug and should something be normalized here?

**Comment by cbespin** [comment]
Good catch, thanks! I will look into it, I have no experience with discrete convolutions (yet) and did not write that code.
